src/views/News/components/NewsItem/newsItem.scss

```diff
@@ -58,3 +58,14 @@
 .unapproved .news-item:hover .news-heading {
     color: $neutral-dark-gray;
 }
+
+// Prepending "button." increases the specificity of the selector
+// and allows override of MUI theme
+button.btn {
+    margin: 0.5rem;
+}
+
+button.deleteButton {
+    border-color: $secondary-red;
+    color: $secondary-red;
+}
```

Comment:
Wow, I never knew you could do this! I will definitely try to give this a go next time I style. This hopefully might help me avoid spending a lot of time converting a Class component into a functional component( aka hooks) to use Material-UI's styling functions.